tests/kuttl/tests/openstackcontrolplane_scale/04-scale_down_openstackcontrolplane.yaml

```diff
@@ -6,11 +6,13 @@ apiVersion: kuttl.dev/v1beta1
 kind: TestStep
 commands:
   - command: |
-      oc annotate vm controller-0 osp-director.openstack.org/openstackvmset-delete-virtualmachine=true
+      oc annotate vm controller-0 osp-director.openstack.org/delete-host=true
     namespaced: true
   - command: |
-      oc annotate vm controller-1 osp-director.openstack.org/openstackvmset-delete-virtualmachine=true
+      oc annotate vm controller-1 osp-director.openstack.org/delete-host=true
     namespaced: true
   - command: |
       oc patch openstackcontrolplane overcloud --type='json' -p='[{"op": "replace", "path": "/spec/virtualMachineRoles/controller/roleCount", "value":1}]'
-    namespaced: true
\ No newline at end of file
+    namespaced: true
+  - command: |
+      sleep 5
```

Comment:
This is to give the controller a few moments to reconcile.  I've seen a rare case where this step is satisfied, but then a second later the controllers do something that should have caused a failure.